src/translation.jl

```diff
@@ -183,7 +176,30 @@ function translate_formula(name::AbstractString; verbose::Bool=false)
 
     # Resynthesize the bottle stanza and embed it into `formula` once more
     bottle_stanza = join(bottle_lines, "\n")
-    formula = formula[1:m.offset] * bottle_stanza * formula[m.offset+length(m.match):end]
+    formula = formula[1:m.offset-1] * bottle_stanza * formula[m.offset+length(m.match):end]
+
+    # Find any depends_on lines, substitute in any translated formulae
+    adjustment = 0
+    for m in eachmatch(r"depends_on\s+\"([^ ]+)\"", formula)
+        # This is the path that this dependency would have if it has been translated
+        dep_name = m.captures[1]
+        auto_dep_path = joinpath(auto_tappath, "$(dep_name).rb")
+
+        # If this dependency has been translated, then prepend "staticfloat/juliatranslated"
+        # to it in the formula, so there is no confusion inside of Homebrew
+        if isfile(auto_dep_path)
+            if verbose
+                println("translation: replacing dependency $dep_name because it's been translated before")
+            end
+            new_name = tapname * dep_name
```

Comment:
Shoot, I had it hardcoded in, then did a "replace-all", and didn't realize that I had screwed up `tapname` vs. `auto_tapname`.